# error-discovery
调用自己创建的类时报错
